lib/commands/generate.js

```diff
@@ -39,7 +39,10 @@ function genSidebar(cwdPath, sidebarPath) {
     pathname = pathname.replace(cwdPath + '/', '')
     let filename = path.basename(pathname, '.md')
     let splitPath = pathname.split(path.sep)
-
+    pathname = pathname.replaceAll(' ','%20')
+      .replaceAll('?','%3F')
+      .replaceAll(')','%29')
+      .replaceAll('#','%23')
```

Comment:
```suggestion
    pathname = pathname.replaceAll(' ','%20')
      .replaceAll('?', '%3F')
      .replaceAll(')', '%29')
      .replaceAll('#', '%23')
```

TypeError: pathname.replaceAll is not a function